src/Pillar-Chrysal/ConfigurationForPillar.class.st

```diff
@@ -384,6 +384,11 @@ ConfigurationForPillar >> declaredItems [
 	^ #( #bibFile #title #attribution #headingLevelOffset #series #keywords #outputDirectory #baseDirectory #outputFile #mainDocument #inputFile #latexTemplate #scrambledEmailAddresses #separateOutputFiles #renderStructureAsSlide #latexChapterTemplate #htmlTemplate #htmlChapterTemplate #newLine #latexWriter #htmlWriter #slideInTemplateForDeckJS #defaultScriptLanguage #levelSpecification #levels #tocFile )
 ]
 
+{ #category : #'as yet unclassified' }
```

Comment:
This should be categorized

Reply:
Categorized in tests (as in Pillar-Tests-ExportHTML)